doc/about.rst

```diff
@@ -91,6 +93,16 @@ Environment also funds several students to work on the project part-time.
 .. image:: images/nyu_short_color.png
    :width: 200pt
    :align: center
+   :target: http://cds.nyu.edu/mooresloan/
+
+`Télécom Paristech <http://www.telecom-paristech.com>`_ funds Manoj Kumar (2014),
+Tom Dupré la Tour (2015), Thierry Guillemot (2016) and Raghav R V (2016) to
+work on scikit-learn.
+
+.. image:: _static/img/telecom.png
```

Comment:
No it's my fault. This file is generated automatically with sphinx but it doesn't exist before the check is complete.